### PR TITLE
hub: add example to raise PR against a specific branch

### PR DESCRIPTION
**Pull request by mfrw**
- hub: add example to raise PR against a specified branch

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Closes: #7470

### CODE REVIEW

File: pages/common/gh.md [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
 `gh pr create`
 
-- View a pull request in the browser:
+- Create a pull request to a specified branch instead of the default:
 
-`gh pr view --web {{pr_number}}`
+`gh pr create --base {{dev}}`
```
Comment (navarroaxel) [inline pages/common/gh.md:29]: I think this is an specific example, and we have the pages `gh-pr` and `gh-pr-create` for more specific arguments.

Comment (mfrw) [reply pages/common/gh.md:29]: Apologies, did not look :(, let me update

File: pages/common/hub.md [hunk 2]
```diff
@@ -20,6 +20,10 @@
 
 `hub pull-request --no-edit`
 
+- Create a PR against a specified branch instead of the default branch:
+
+`hub pull-request --base {{dev}}`
```
Comment (navarroaxel) [inline pages/common/hub.md:25]: I think we should use the same placeholder as `gh-pr-create`:
```suggestion
`hub pull-request --base {{base_branch}}`
```

Comment (mfrw) [reply pages/common/hub.md:25]: Sure, we can do that as well, but since we do not have a page for it ? Should we probably add one `hub-pull-request` ?

**Review by einverne — APPROVED** [review]
Looks good to me.

File: pages/common/hub.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/hub.md:25]: Yeah, add it.

**Review by sbrl — APPROVED** [review]
Looks ok to me after @navarroaxel's comment is addressed.

**Comment by sbrl** [comment]
Waiting on OP to update PR.

**Review by mfrw — COMMENTED** [review]
Closing this PR, as per the discussion, a new page should be added.

**Pull request closed without merging**